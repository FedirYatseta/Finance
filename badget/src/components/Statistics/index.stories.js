import React from 'react';

import Statistic from '.';

export default {
    title: 'Example/Statistic',
    component: Statistic,
};
const Template = (args) => <Statistic {...args}></Statistic>;

export const Income = Template.bind({});
Income.args = {

};