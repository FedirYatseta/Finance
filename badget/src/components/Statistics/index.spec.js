

//import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';
import Statistics from '.';



describe('Statistics', () => {
  it('should show Statistics', () => {
    const component = renderer.create(
      <Statistics />,
    );
    expect(component).toMatchSnapshot();

  })
});