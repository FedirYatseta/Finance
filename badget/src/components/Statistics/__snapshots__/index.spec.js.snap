// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Statistics should show Statistics 1`] = `
<h1>
  Statistic
</h1>
`;
